lib/pure/os.nim

```diff
@@ -2009,8 +2009,8 @@ proc removeFile*(file: string) {.rtl, extern: "nos$1", tags: [WriteDirEffect], n
   if not tryRemoveFile(file):
     raiseOSError(osLastError(), file)
 
-proc tryMoveFSObject(source, dest: string): bool {.noWeirdTarget.} =
-  ## Moves a file or directory from `source` to `dest`.
+proc tryMoveFSObject(source, dest: string, isMoveDir: bool): bool {.noWeirdTarget.} =
+  ## Moves a file (or directory if `isMoveDir` is true) from `source` to `dest`.
   ##
   ## Returns false in case of `EXDEV` error.
```

Comment:
i see you've udpated the comment above, which is good, but that comment mentioning EXDEV also needs to be updated (for windows, it returns false if isMoveDir and `AccessDeniedError = OSErrorCode(5)` which can be obtained for dir moves across partitions)

Reply:
Oh my god, thanks for being this patient with me ^^